### DISCUSSION TITLE
Design: App Management & App Hub

### DISCUSSION THREAD

**Discussion by cooper-joe** [dhis2/notes#225] [Ideas]
Here’s the [ongoing design work for the App Hub and App Management app](https://www.sketch.com/s/fbbe8e47-3ca6-4b66-a42f-6ca413613e91/a/JnAr1M7/play), all still in progress, but ready to start discussing the feature set and direction. The included features are based on the outcome of [this discussion](https://github.com/dhis2/notes/discussions/200#discussioncomment-317811).

**Comment by Philip-Larsen-Donnelly** [comment]
I would like to clarify if and how we plan to manage the quality of apps over their lifetime?
For instance, we are currently applying a quality gate (review board) for admission to the App Hub, but what happens afterwards?
Do we review new versions to different channels, or just stable channel, or not at all? What if a new version introduces changes that we would not have approved for the App Hub? Likewise, if an app is recommended, or competition winner, several versions later that might not be the case any longer.

Will we simply rely on, and react to, the "report a problem..."/ "flag this app" feedback? (I don't see these in the mockups yet).

**Reply by cooper-joe** [reply]
![image](https://user-images.githubusercontent.com/33054985/107956035-d987be80-6f9e-11eb-8c99-61b209f01905.png)

"Report a problem" is included on the App Hub app details page, but would also be included in App Management too (have just updated the mockup to reflect this). I'm not sure if it's too subtle in the sub-actions in the right sidebar, we may want to elevate it to a more important action if we want to encourage its use.
